src/components/ControlPanel.tsx

```diff
@@ -430,6 +454,16 @@ Are you sure you want to continue?`,
                         )}
                     </>
                 </Toggle>
+                <Toggle
+                    onToggle={() =>
+                        dispatch(settingsActions.toggleAutoUpdateOBFirmware())
+                    }
+                    isToggled={autoUpdateOBFirmware}
+                    label="Auto update OB Firmware"
```

Comment:
```suggestion
                    label="OB Firmware automatic update"
```

*Please also apply the same rule to line 448 (Automatic reset) and line 436 (Automatic memory read)*